src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,40 @@
+{
+  /* DON'T use conditional rendering to hide the notification */
+}
+
+{
+  /* Add the 'hidden' class to hide the message smoothly */
+}
+
+import React, { useContext } from 'react';
+import { ErrorContext } from './TodoContext';
+import classNames from 'classnames';
+
+type Props = {};
+
+export const ErrorNotification: React.FC<Props> = () => {
+  const { errorMessage, setErrorMessage } = useContext(ErrorContext);
```

Comment:
```suggestion
export const ErrorNotification = () => {
  const { errorMessage, setErrorMessage } = useError();
```
- If you don't have props you don't need to specify the type
- Create hoooks for contexts as I mentioned previously